sprotty-vscode-webview/package.json

```diff
@@ -40,13 +40,14 @@
   "types": "lib/index",
   "dependencies": {
     "reflect-metadata": "^0.1.12",
-    "sprotty": "0.9.0",
+    "sprotty": "0.10.0",
     "sprotty-vscode-protocol": "^0.0.5",
     "vscode-uri": "2.1.1"
   },
   "devDependencies": {
     "rimraf": "latest",
     "tslint": "^5.11.0",
+    "file-loader": "6.2.0",
```

Comment:
I think this dependency is not necessary here.

Reply:
Right, but it is necessary in the example webview because of its webpack configuration due to some sprotty change in 0.10. Moved it there.